java/spacewalk-java.changes

```diff
@@ -1,3 +1,4 @@
+- Enable Japanese and Korean language support
```

Comment:
Since this is only for SUSE Manager (Uyuni had it already), not sure if we should mention it.